packages/evolution-common/src/services/conditionals/__tests__/checkConditionals.test.ts

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import each from 'jest-each';
+import { checkConditionals } from '../checkConditionals';
+
+const interview = {
+    responses: {
+        _activePersonId: 'bb6c33d8-5b68-4eb4-a031-2ffad0fd2ba5',
+        _isNull: null,
+        _isArray: ['a', 'b', 'c'],
+        _isEmptyArray: [],
+        _isString: 'a',
+        _isTrueBoolean: true,
+        _isNumber1: 1,
+        household: {
+            size: 1,
+            persons: {
+                'bb6c33d8-5b68-4eb4-a031-2ffad0fd2ba5': {
+                    age: 33,
+                }
+            }
+        }
+    }
+};
+
+each([
+    [
+        'Path exist, and the value is the same as the interview responses',
+        [
+            {
+                path: '_activePersonId',
+                comparisonOperator: '===',
+                value: interview.responses._activePersonId
+            }
+        ],
+        [true, null]
+    ],
+    [
+        'Path exist, but the value is different from the interview responses',
+        [
+            {
+                path: '_activePersonId',
+                comparisonOperator: '!==',
+                value: interview.responses._activePersonId
+            }
+        ],
+        [false, null]
+    ],
+    [
+        'Path exist, but the value is different from null',
+        [
+            {
+                path: '_isNull',
+                comparisonOperator: '!==',
+                value: 'null'
+            }
+        ],
+        [false, null]
+    ],
+    [
+        'Path exist, but the value is null',
```

Comment:
C'est lexical, et peut-être plus nécessaire si tu changes les titres des test, mais 'but' sous-entend que ça devrait être `false`, alors que le 'and' sous-entend `true`.